app/decorators/user_decorator.rb

```diff
@@ -100,11 +100,15 @@ def verified_account_partial
 
   private
 
+  def model
+    __getobj__
```

Comment:
methinks the whole point of this change is to be able to call `User` methods on a `DecoratedUser` with invisible delegation. So a POR class won't do that afaik.